Apps/W1/ShopifyConnector/app/Products/Codeunits/ShpfyProductMapping.Codeunit.al

```diff
@@ -0,0 +1,275 @@
+/// <summary>
+/// Codeunit Shpfy Product Mapping (ID 30181).
+/// </summary>
+codeunit 30181 "Shpfy Product Mapping"
+{
+    Access = Internal;
+
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"